### ISSUE TITLE
508-defect-2: Labels for names must be descriptive 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#42250]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

The claimant information page and the deceased Veteran information page both ask for "your name" without clarification as _whose_ name that is referring to. This emerges again on step 3 for other names the Veteran used.

## Acceptance Criteria

- [ ] "Your [x] name" under claimant information is rewritten to "Claimant's [x] name" 
- [ ] First paragraph under step 1 of 6 is rewritten to: "As the person claiming burial benefits, you need to provide some key information. You aren’t required to fill in all fields, but we can review your application faster if you provide more information."
- [ ] "Your [x] name" under the deceased Veteran's information is rewritten to "Veteran's [x] name"

## Steps to Recreate

1. Enter `[<IP ADDRESS>](https://staging.va.gov/burials-and-memorials/application/530/claimant-information)` in browser
2. Start the form without signing in
3. View the claimant and deceased Veteran pages and confirm both ask for "your" name
4. On step 3, for the question "Did the Veteran serve under another name?" page, answer "no" 

## Proposed Solution (if known)

Use specific labels for both the claimant's page and the deceased Veteran's page.

<img width="358" alt="Screen Shot 2022-06-02 at 3 52 38 PM" src="https://user-images.githubusercontent.com/14154792/171726562-2709367b-8736-4f7a-9e33-953d43f3fe17.png">

<img width="358" alt="Screen Shot 2022-06-02 at 3 53 46 PM" src="https://user-images.githubusercontent.com/14154792/171726718-ea75ee4e-ec7c-49dc-9ab3-f439e09cce98.png">

**Comment by jsokohl** [comment]
For the second step, can we make it conditional? That is, if the claimant is the person asking for benefits, can we skip step 2? That would be sensible.

**Comment by joshkimux** [comment]
@jsokohl , love that train of thought to reduce steps. We may want to rope @Mottie in to help us here though as I believe that [the introduction page explicitly states that the deceased Veteran's information is required](https://staging.va.gov/burials-and-memorials/application/530/introduction) in order for the survivor to claim benefits for burial. If that's not _actually_ required, then we should skip it for sure. @Mottie could you help us out? 🙇

**Comment by jsokohl** [comment]
Sure thing—it’s just that, if the person who is the covered veteran is not deceased, that’s where there’s a problem. For example, if I am filling the form out for myself, I’m the covered person but not yet deceased.  If it’s person A who is filling it out for person B who IS deceased, then the form is accurate and sufficiently clear as is. Right?

HTH. 

joe
Joe Sokohl
1.804.873.6964

**Comment by jsokohl** [comment]
Glad to do that when I get back from PTO next week, or have y’all run with it!

joe
Joe Sokohl
1.804.873.6964

**Comment by joshkimux** [comment]
Got a quick response from @mnorthuis ❤️ 🙇 

> For burial allowance, you have to provide the Veteran's death certificate and date of death, so a Veteran cannot complete this application for themselves.

**Comment by mnorthuis** [comment]
Family members can have accounts too, and it allows save in progress.  It should, however, only pre-populate info on the claimant, and not the Veteran.

**Comment by joshkimux** [comment]
@jsokohl couple of updates for you now that you're back, hope you had a great break away from work 🥳  (and apologies for tagging you earlier when you were on PTO, won't do that again) 🙇 

Does Mikki's response answer your initial questions?

**Comment by jsokohl** [comment]
All good as spec'ed here, I'd say: 
1. I would edit the first instruction to "As the person claiming burial benefits, you need to provide some key information. You aren’t required to fill in all fields, but we can review your application faster if you provide more information." This helps with traceability with the word "claimant."
2. I would say that the labels for step 2 could be merely "Veteran's first name," "Veteran's last name," and so on. The context is telegraphed by the heading **Decease Veteran information**. We don't have to repeat the word "deceased."

**Comment by joshkimux** [comment]
Sounds great to me Joe! @jerekshoe if Mikki gives us the clear let's push this through. I've updated the acceptance criteria with Joe's suggestions in the meantime 😄

**Comment by bethpottsVADEPO** [comment]
@RLHecht tagging you on this ticket.

**Comment by jerekshoe** [comment]
This has already been merged to main. I can revert it if you would like

**Comment by RLHecht** [comment]
Yes, please @jerekshoe . We have one content change we'd like you to make. Under **Step 1 of 6: Claimant information** the text should read:

As the person claiming burial benefits, you'll need to provide some information. You don't need to fill in all the fields, but we can review your application faster if you provide more information.

**Comment by jerekshoe** [comment]
If we make the change to the description text, is this ready to go then? If that's the case it would be easier to just make another PR to make the change instead of reverting the previous one

**Comment by RLHecht** [comment]
Yeah, this is our only change so however you think it's easiest to handle is OK with me.

**Comment by jerekshoe** [comment]
@RLHecht ok great. Sorry about that, I had misread the previous comments and thought that this was good to go when I originally merged it.

**Comment by jsokohl** [comment]
Awesome! And a great edit.

Joe
